docs/design-docs/07-package-orchestration.md

```diff
@@ -0,0 +1,941 @@
+```
+
+All command line flags supported:
+
+* `--directory` - Directory within the repository where to look for packages.
+* `--branch` - Branch in the repository where finalized packages are committed.
+  (defaults to `main`)
+* `--name` - Name of the package repository Kubernetes resource. If unspecified,
+  will default to the name portion (last segment) of the repository URL
+  (`blueprint` in the example above)
+* `--description` - Brief description of the package repository.
+* `--deployment` - Boolean value; If specified, repository is a deployment
+  repository; published packages in a deployment repository are considered
+  deployment-ready.
+* `--repo-username` - Username for repository authentication.
+* `--repo-password` - Password for repository authentication.
+
+Additionally, common `kubectl` command line flags for controlling aspects of
+interaction with the Kubernetes apiserver, logging, and more (this is true for
+all `kpt` CLI commands which interact with Porch).
+
+Use the `kpt alpha repo get` command to query registered repositories:
+
+```sh
+# Query registered repositories
+$ kpt alpha repo get
+
+NAME         TYPE  CONTENT  DEPLOYMENT  READY  ADDRESS
+blueprints   git   Package              True   https://github.com/platkrm/blueprints.git
+deployments  git   Package  true        True   https://github.com/platkrm/deployments.git
+```
+
+The `kpt alpha <group> get` commands support common `kubectl`
+[flags](https://kubernetes.io/docs/reference/kubectl/cheatsheet/#formatting-output)
+to format output, for example `kpt alpha repo get --output=yaml`.
+
+The command `kpt alpha repo unregister` can be used to unregister a repository:
+
+```sh
+# Unregister a repository
+$ kpt alpha repo unregister deployments
+```
+
+### Package Discovery And Introspection
+
+The `kpt alpha rpkg` command group contains commands for interacting with
+packages managed by the Package Orchestration service. the `r` prefix used
+in the command group name stands for 'remote'.
+
+The `kpt alpha rpkg get` command list the packages in registered repositories:
+
+```sh
+# List package revisions in registered repositories
+$ kpt alpha rpkg get
+
+NAME                                                 PACKAGE  REVISION  LATEST  LIFECYCLE  REPOSITORY
+blueprints-0349d71330b89ee48ac85167598ef23021fd0484  basens   main      false   Published  blueprints
+blueprints-2e47615fda05664491f72c58b8ab658683afa036  basens   v1        true    Published  blueprints
+blueprints-7e2fe44bfdbb744d49bdaaaeac596200102c5f7c  istions  main      false   Published  blueprints
+blueprints-ac6e872be4a4a3476922deca58cca3183b16a5f7  istions  v1        false   Published  blueprints
+blueprints-421a5b5e43b03bc697d96f471929efc6ba3f54b3  istions  v2        true    Published  blueprints
+...
+```
+
+The `LATEST` column indicates whether the package revision is the latest among
+the revisions of the same package. In the output above, v4 is the latest
```

Comment:
Should it be `v2` instead of `v4` here? There doesn't seem to be a `v4` in the output.

Reply:
wow, great catch! fixed. thank you!